week05/my-todolist/src/App.js

```diff
@@ -1,32 +1,42 @@
-import { useEffect, useState } from "react";
-import Header from "./Header";
-import TodoList from "./TodoList";
-import TodoAdd from "./TodoAdd";
-import StateTest from "./StateTest";
+import { Routes, Route, NavLink } from "react-router-dom";
+//import StateTest from "./StateTest";
 import "./App.css";
-
-const TODO_LIST_KEY = "todoList";
+import MyTodolist from "./components/pages/MyTodolist";
+import MyPlaylist from "./components/pages/MyPlaylist";
 
 function App() {
-  const [todoList, setTodoList] = useState(() => {
-    const savedTodoList = localStorage.getItem(TODO_LIST_KEY);
-    return savedTodoList ? JSON.parse(savedTodoList) : [];
-  });
-
-  useEffect(() => {
-    localStorage.setItem(TODO_LIST_KEY, JSON.stringify(todoList));
-  }, [todoList]);
-
-  // 임시 state 생성
-  const [tempState, setTempState] = useState(0);
-
+  const style = {
```

Comment:
6. `document.body.style.backgroundImage`의 값을 state `bgImg`로 설정합니다.
7. state `bgImg`에 변경사항이 생겼을 때만 업데이트되도록, useEffect로 감싸줍니다.

```suggestion
  useEffect(() => {
    document.body.style.backgroundImage = `url(${bgImg})`;
  }, [bgImg]);

  const style = {
```